Document-API.md

```diff
@@ -2,20 +2,167 @@
 
 The documents entry point is used to access the data for documents, as opposed to metadata (which is found in collections).  The representation of a document is up to the implementation.
 
+## Default Scheme
+
+- The document API can support as many response format as the content provider wishes.
+- The document API must support an `application/tei+xml` response format that is made of at least the following structure : 
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+  <dts:fragment xmlns:dts="DTS PURL ?">
+    <!-- XML or string of the passage requested here -->
+  </dts:fragment>
+</TEI>
+```
+- The rest of the document can contain any node of the TEI
+- The requested passage should be part of the `dts:fragment` Node
+
+## URI 
+
 ### Query Parameters
 
 The documents endpoint supports the following query parameters:
 
 | name | description                              | methods |
 |------|------------------------------------------|---------|
 | id   | identifier for a document |  GET    |
-| q    | full-text search on document data             |  GET    |
-| passage | passage identifier (used together with `id` | n/a    |
-| collection | used with `q` to specify a collection to be searched | n/a |
+| passage | passage identifier (used together with `id`) | n/a    |
+| start | (For range) Start of the passage we want descendants of | GET |
+| end |  (For range) End of the passage we want descendants of | GET |
+
+### Response Headers
+
+The response contains the following response headers:
+
+| name | description |
+|------|-------------|
+| Link | Gives relation to next and previous pages |
+| Content-Type | Content type of the response (By default : `application/tei+xml`)|
+
+### URI Template
+
+Here is a template of the URI for Collection API. The route itself (`/dts/api/navigation/`) is up to the implementer.
```

Comment:
why does this use a different example route than the examples below (/dts/api/navigation vs /dts/api/documents)

Reply:
Just me making a typo... Correcting it right away.